src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -558,6 +558,8 @@ public void EditorButtonPressed()
 
         EnsureGameIsUnpausedForEditor();
 
+        stage.RecordPlayerReproduction();
```

Comment:
This shouldn't happen here as the stage can still prevent the player from switching to the editor if preconditions are no longer met.